### PR TITLE
Add LuxManga as MadaraSource

### PR DESCRIPTION
**Pull request by ntbm**
Add LuxManga as new Source based on Madara to List of Extensions.

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -496,6 +496,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("مانجا عرب تيم Manga Arab Team", "https://mangaarbteam.com", "ar", className = "MangaArabTeam", overrideVersionCode = 1),
         SingleLang("مانجا ليك", "https://mangalek.com", "ar", className = "Mangalek", overrideVersionCode = 1),
         SingleLang("مانجا لينك", "https://mangalink.io", "ar", className = "MangaLinkio", overrideVersionCode = 2),
+        SingleLang("LuxManga", "https://luxmanga.com", "en"),
```
Comment (AntsyLich) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:499]: NO Place it in the write place not at the bottom

Comment (ntbm) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt]: Fixed

**Comment by Riztard** [comment]
maybe just combine with the other madara pr

**Comment by ntbm** [comment]
> maybe just combine with the other madara pr 

imho I like it more when PRs are narrow in Scope. I rather merge the changes into my other PR.
Don't know how it is handled around here

**Pull request merged**